python/lsst/sdm/schemas/dp1.yaml

```diff
@@ -5921,424 +5921,558 @@ tables:
   - name: nDiaSources
     description: Number of diaSources associated with this diaObject.
     datatype: long
+    # ivoa:unit: '1'
   - name: radecMjdTai
     description: Time at which the object was at a position ra/dec, expressed as Modified
       Julian Date, International Atomic Time.
     datatype: double
+    ivoa:unit: d
   - name: g_psfFluxLinearSlope
     description: Slope of a linear model fit to diaSource PSF flux vs time
     datatype: double
+    ivoa:unit: nJy/d
```

Comment:
For the record, it would be nice to have a reference confirming the units (mainly the denominator) but we can go ahead with this and `*_psfFluxMaxSlope` on the basis that they match `apdb.yaml`.